src/domain/about/__tests__/AboutPage.test.tsx

```diff
@@ -1,27 +1,73 @@
 import i18n from 'i18next';
 import React from 'react';
+import wait from 'waait';
 
+import { AboutPagesDocument } from '../../../generated/graphql';
+import { fakeAboutPages } from '../../../util/mockDataUtils';
 import { act, render, screen } from '../../../util/testUtils';
 import AboutPage from '../AboutPage';
 
-it('should show about page in correct language', async () => {
-  render(<AboutPage />);
+const fakeAboutPagesResponse = fakeAboutPages(1);
 
+const mocks = [
+  {
+    request: {
+      query: AboutPagesDocument,
+    },
+    result: {
+      data: {
+        aboutPages: fakeAboutPagesResponse,
+      },
+    },
+  },
+];
+
+test('should render about page', async () => {
+  render(<AboutPage />, { mocks });
+
+  await act(wait);
   act(() => {
     i18n.changeLanguage('fi');
   });
 
-  expect(screen.getAllByText('Tietoa palvelusta')).toHaveLength(1);
+  expect(
+    screen.getAllByText(
+      fakeAboutPagesResponse.data[0].headingSection?.fi as string
+    )
+  ).toHaveLength(1);
```

Comment:
is there a reason to not use screen.getByText()?

`expect(
    screen.getByText(
      fakeAboutPagesResponse.data[0].headingSection?.fi as string
    )
  ).toBeInTheDocument()`

Reply:
if not, I would change this to make it simpler

Reply:
Simplified tests